src/meson.build

```diff
@@ -120,3 +120,11 @@ endif
     ],
     install: true,
   )
+
+# Set CAP_SYS_NICE on install
+if cap_dep.found() and (not get_option('skip_setcap'))
+  setcap = find_program('setcap', '/usr/sbin/setcap', '/sbin/setcap', required : false)
+  if setcap.found()
+    meson.add_install_script('setcap.sh', setcap, get_option('bindir'), exe.name(), skip_if_destdir: true)
```

Comment:
skip_if_destdir is wrong here. In DESTDIR installations, you still want to run setcap because it will be tracked by (some) package managers. You are anyways checking if you have root in order to do it.

Aside: why so many ways to find the setcap binary? Shouldn't $PATH be quite good enough? It should anyways be available in the root contexts where you want to run it.

Reply:
Curious what those package managers are ?  Everything I saw suggested it wouldn't be automatically tracked.  The `find_program` statement came from the example in the meson manual, my guess is it covers  cases where a non-root user installs to a privileged location (so it wouldn't be in the user's path at config/build time).